### ISSUE TITLE
Color code the output

### ISSUE THREAD

**Issue by kenden** [hadolint/hadolint#267]
- [] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

In order to make the errors more obvious, it would be nice to color them in red in the output.
Like shellcheck does:

![image](https://user-images.githubusercontent.com/480586/44208550-f25e5180-a160-11e8-99f6-c10ab4364d96.png)

Then as a user, it's quicker to see a problem.

### Actual behavior
All the results are in black and white.
Errors are not quick to spot.

### Steps to reproduce the behavior

```bash
$ echo 'FROM ubuntu:latest' > Dockerfile
$ hadolint Dockerfile
Dockerfile:1 DL3007 Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag
```

### Version
```bash
$ hadolint --version
Haskell Dockerfile Linter 1.12.0-no-git

$ uname -a
Darwin rectacted-mac07.local 17.7.0 Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64 x86_64 i386 MacBookPro11,3 Darwin
```

**Comment by kenden** [comment]
Maybe the colors could be used like they are in shellcheck:
https://github.com/koalaman/shellcheck/blob/f91b5bc2705b48d6436e92076dcf77866edcb518/src/ShellCheck/Formatter/TTY.hs#L38

In the example below, 
all the lines could be red.
or just the part about the error code (ex DL3009)?

When there is a lot of output, it would make it easier to find the errors.

```bash
$hadolint foldername/Dockerfile
foldername/Dockerfile:6 DL3009 Delete the apt-get lists after installing something
foldername/Dockerfile:6 DL3015 Avoid additional packages by specifying `--no-install-recommends`
foldername/Dockerfile:6 DL3014 Use the `-y` switch to avoid manual input `apt-get -y install <package>`
foldername/Dockerfile:98 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:100 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:107 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:109 SC2185 Some finds don't have a default path. Specify '.' explicitly.
foldername/Dockerfile:109 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:126 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:131 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:136 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:141 SC2185 Some finds don't have a default path. Specify '.' explicitly.
foldername/Dockerfile:141 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:147 SC1117 Backslash is literal in "\(". Prefer explicit escaping: "\\(".
foldername/Dockerfile:147 SC1117 Backslash is literal in "\)". Prefer explicit escaping: "\\)".
foldername/Dockerfile:147 SC1117 Backslash is literal in "\1". Prefer explicit escaping: "\\1".
foldername/Dockerfile:147 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:152 SC1117 Backslash is literal in "\(". Prefer explicit escaping: "\\(".
foldername/Dockerfile:152 SC1117 Backslash is literal in "\)". Prefer explicit escaping: "\\)".
foldername/Dockerfile:152 SC1117 Backslash is literal in "\1". Prefer explicit escaping: "\\1".
foldername/Dockerfile:152 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:157 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:158 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:161 SC2015 Note that A && B || C is not if-then-else. C may run when A is true.
foldername/Dockerfile:161 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:162 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:166 DL3010 Use ADD for extracting archives into an image
foldername/Dockerfile:167 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:173 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:174 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:181 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:182 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:189 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:190 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:194 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:200 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:206 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:212 DL3003 Use WORKDIR to switch to a directory
foldername/Dockerfile:226 DL3003 Use WORKDIR to switch to a directory
```

**Comment by lorenzo** [comment]
Ok, thanks. I think I understand what you mean now :)

**Comment by mperezi** [comment]
I wrote a little Python wrapper for this:

https://github.com/mperezi/hadolint-wrapper

It would be nice if this could be shipped into a future hadolint release @lorenzo

**Comment by lorenzo** [comment]
I'm accepting pull requests for this feature, I currently don't have the bandwidth to work on this one. Thanks for that wrapper @mperezi 👍
